kubectl-plugin/src/resources/utils.rs

```diff
@@ -60,11 +60,32 @@ impl From<&str> for OutputFormat {
     }
 }
 
-pub fn print_table<T>(output: &OutputFormat, obj: Vec<T>)
+impl<T> CreateRows for Vec<T>
+where
+    T: CreateRows,
+{
+    fn create_rows(&self) -> Vec<Row> {
+        let mut rows = vec![];
+        self.iter().for_each(|i| rows.extend(i.create_rows()));
+        rows
+    }
+}
+
+// GetHeaderRow trait to be implemented by Volume/Pool to fetch the corresponding headers.
+impl<T> GetHeaderRow for Vec<T>
+where
+    T: GetHeaderRow,
+{
+    fn get_header_row(&self) -> Row {
+        self[0].get_header_row()
```

Comment:
this would panic on empty lists, so I guess it could be something like this:
self.get(0).map(GetHeaderRow::get_header_row).unwrap_or_default()